packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -468,42 +472,44 @@ describe("Garbage Collection Tests", () => {
 
         beforeEach(async () => {
             // Set up the reference graph such that all nodes are referenced. Add in a couple of cycles in the graph.
+            // 0 - 1 - 2 - 3
+            //      \- 0    \- 0
```

Comment:
What does this mean?

Reply:
I think it might be better to draw the lines back to the 0 node from the 1 node and the 3 node.

Reply:
Haha it was just to help me, if it's so unclear I'll remove it.

Also, we should rename the string values to 0-3 not 1-4, it's super confusing when debugging a test :P

Reply:
Ok I tried Tyler's idea.  Someone can delete it in a future PR if you don't like it :)